Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivity.java

```diff
@@ -34,7 +38,9 @@ public final class SettingsActivity extends AppCompatActivity implements Setting
   private static final String FRAGMENT_TAG = "settings";
   private SettingsActivityPresenter mPresenter;
 
-  private ProgressDialog dialog;
+  private AlertDialog dialog;
+
+  public CollapsingToolbarLayout mToolbarLayout;
```

Comment:
Instead of making this public, please provide a setter for the title and expose it through SettingsActivityView.